drivers/spi/spi_andes_atcspi200.c

```diff
@@ -398,43 +401,41 @@ static void spi_atcspi200_irq_handler(void *arg)
 #define SPI_ROM_CFG_XIP(node_id) false
 #endif
 
-#define SPI_INIT(n)								\
-	static struct spi_atcspi200_data spi_atcspi200_dev_data_##n = {		\
-		SPI_CONTEXT_INIT_LOCK(spi_atcspi200_dev_data_##n, ctx),		\
-		SPI_CONTEXT_INIT_SYNC(spi_atcspi200_dev_data_##n, ctx),		\
-		SPI_CONTEXT_CS_GPIOS_INITIALIZE(DT_DRV_INST(n), ctx)		\
-		SPI_IF_NO_CMD(n)						\
-		SPI_BUSY_INIT							\
-		SPI_DMA_CHANNEL(n, rx, RX, PERIPHERAL, MEMORY)			\
-		SPI_DMA_CHANNEL(n, tx, TX, MEMORY, PERIPHERAL)			\
-	};									\
-	static void spi_atcspi200_cfg_##n(void);				\
-	static const struct spi_atcspi200_cfg spi_atcspi200_dev_cfg_##n = {	\
-		.cfg_func = spi_atcspi200_cfg_##n,				\
-		.base = DT_INST_REG_ADDR(n),					\
-		.irq_num = DT_INST_IRQN(n),					\
-		.f_sys = DT_INST_PROP(n, clock_frequency),			\
-		.xip = SPI_ROM_CFG_XIP(DT_DRV_INST(n)),				\
-	};									\
-										\
-	DEVICE_DT_INST_DEFINE(n,						\
-		spi_atcspi200_init,						\
-		NULL,								\
-		&spi_atcspi200_dev_data_##n,					\
-		&spi_atcspi200_dev_cfg_##n,					\
-		POST_KERNEL,							\
-		CONFIG_SPI_INIT_PRIORITY,					\
-		&spi_atcspi200_api);						\
-										\
-	static void spi_atcspi200_cfg_##n(void)					\
-	{									\
-										\
-		IRQ_CONNECT(DT_INST_IRQN(n),					\
-			    DT_INST_IRQ(n, priority),				\
-			    spi_atcspi200_irq_handler,				\
-			    DEVICE_DT_INST_GET(n),				\
-			    0);							\
-										\
-	}
+#define SPI_INIT(n)							\
+	static struct spi_atcspi200_data spi_atcspi200_dev_data_##n = {\
```

Comment:
```suggestion
	static struct spi_atcspi200_data spi_atcspi200_dev_data_##n = { \
```

Reply:
Thanks, that is great!
I update the source code and rebase it.